linux/ci_configure_app.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```

Comment:
This file requires better name. 
`ci_configure_wrappture_build`?

Reply:
no, it configure all apps. not just wrappture app.

Reply:
ok, change it to `ci_configure_apps` then

Reply:
I change to ci_configure_apps